src/main.tsx

```diff
@@ -1,10 +1,29 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
+import { RouterProvider, createRoutesFromElements } from 'react-router'
+import { Route, createBrowserRouter } from 'react-router-dom'
 import App from './App.tsx'
+import Bookmarked from './Pages/Bookmarked.tsx'
+import Categories from './Pages/Categories.tsx'
+import CategorySpecific from './Pages/CategorySpecific.tsx'
+import FilmView from './Pages/FilmView.tsx'
+import HomePage from './Pages/HomePage.tsx'
 import './index.css'
 
-ReactDOM.createRoot(document.getElementById('root')!).render(
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path='/' element={<App />}>
+      <Route index element={<HomePage />} />
+      <Route path='filmview' element={<FilmView />} />
+      <Route path='bookmarked' element={<Bookmarked />} />
+      <Route path='categories' element={<Categories />} />
+      <Route path='categories/genre' element={<CategorySpecific />} />
```

Comment:
Maybe we should make this link dynamic and customized for which category the user chooses to click on? 😃
For instance: categories/:id